addons/mail/static/src/discuss/call/common/meeting_side_actions.js

```diff
@@ -19,32 +19,31 @@ export class MeetingSideActions extends Component {
     setup() {
         this.store = useService("mail.store");
         useSubEnv({ inMeetingSideActions: true });
-        onWillRender(() => {
-            const quickThreadActionIds = ["invite-people", "meeting-chat"];
-            const threadActions = toRaw(this.props.threadActions);
-            const { quick, other, group } = threadActions.partition;
-            const partitionedActions = {
-                quick: quick.filter((action) => !quickThreadActionIds.includes(action.id)),
-                other: other.filter((action) => !quickThreadActionIds.includes(action.id)),
-                group: group
-                    .map((group) =>
-                        group.filter((action) => !quickThreadActionIds.includes(action.id))
-                    )
-                    .filter((g) => g.length > 0),
-            };
-            const actions = threadActions.actions.filter((action) =>
-                quickThreadActionIds.includes(action.id)
-            );
-            actions.push(
-                threadActions.more({
-                    actions: [
-                        partitionedActions.quick,
-                        partitionedActions.other,
-                        ...partitionedActions.group,
-                    ],
-                })
-            );
-            this.actions = actions;
-        });
```

Comment:
This follows the same pattern as in `Message` component, and is necessary to observe state in actions (i.e. `action.badgeText`) while keeping efficient number of renderings.